compiler/rustc_llvm/llvm-wrapper/PassWrapper.cpp

```diff
@@ -1120,9 +1120,15 @@ struct LLVMRustThinLTOData {
 
   // Not 100% sure what these are, but they impact what's internalized and
   // what's inlined across modules, I believe.
+#if LLVM_VERSION_GE(17, 0)
```

Comment:
Oops, we need to start checking against 18 now. I'll fix it in https://github.com/rust-lang/rust/pull/114048.